_data/committers.yml

```diff
@@ -300,6 +300,10 @@
   role: Committer
   alias: rdettai
   affiliation: Cloudfuse
+- name: Remzi Yang
+  role: Committer
+  alias: HaoYang670
```

Comment:
@alamb Could you please update the alias as `remziy`?

Reply:
```suggestion
  alias: remziy
```